seabird_chat_ingest.proto

```diff
@@ -0,0 +1,164 @@
+syntax = "proto3";
+
+option go_package = ".;pb";
+
+import "common.proto";
+
+package seabird;
+
+message ChatUser {
+  string id = 1;
+  string name = 2;
+}
+
+message ChatChannel {
+  string id = 1;
+  string name = 2;
+}
+
+// When a chat service receives a PingChatRequest, it MUST respond with a
+// PongChatEvent containing the copied nonce.
+message PingChatRequest {
```

Comment:
What's the use case of this? If you're heartbeating IRC servers and the like, IMO the chat services themselves should handle that. If this is to heartbeat the actual chat service from core, isn't that already done via the stream?

Reply:
Also, is it reasonable to assume that every chat service can even do this (a heartbeat that is more than `isItDown`)?

Reply:
Doing this allows us to measure latency between core and the process running chat backends... and if those chat backends wanted to pass the same ping request on to their respective chat service, they could. However, yes, there is a heartbeat already on all gRPC server streams, so this may be a bit redundant - I can remove it for the MVP.

Reply:
We can also instrument the RPC layer itself to get latency info. I’d see pings filling a different gap than that even, and I’m not sure what that gap is

Reply:
Ping/pong events were dropped for now so I'm resolving this thread.